src/widgets/Footer/ui/index.tsx

```diff
@@ -1,12 +1,20 @@
+import Box from "@mui/material/Box"
+
 import { Logo } from "~/shared/ui"
+import { Theme } from "~/shared/utils"
 
 import "./style.scss"
 
 const Footer = () => {
   const buildVersion = import.meta.env.VITE_BUILD_VERSION
```

Comment:
Please refactor it into shareable constant

Reply:
Done